src/it/resources/com/google/checkstyle/test/chapter4formatting/rule487modifiers/InputModifierOrder.java

```diff
@@ -8,67 +8,71 @@ strictfp abstract class InputModifierOrder {
   volatile public int whatImReading;
   // violation above ''public' modifier out of order with the JLS suggestions.'
 
-  public volatile boolean sModifierOrderVar = false;
+  public volatile boolean ssModifierOrderVar = false;
 
   /**
    * Illegal order of modifiers for methods. Make sure that the first and last modifier from the JLS
    * sequence is used.
    */
-  strictfp private void doStuff()
+  strictfp private void doStuff() {}
   // violation above ''private' modifier out of order with the JLS suggestions.'
-  {}
```

Comment:
changed curly braces position to make them more readable

Reply:
I am ok, we should not put comments into weird places.